docs/dictionary/command/play.lcdoc

```diff
@@ -27,18 +27,24 @@ play audioClip "Trust No One" looping
 Example:
 play "/usr/local/clips/music.aiff"
 
+Example:
+# to stop the currently playing audio clip
+play stop
+
 Parameters:
+type (enum):
```

Comment:
Done. What about the larger issue of the behavior of play being out of sync with the documentation?